src/app/modules/domain/course/learner-course/learner-course.component.html

```diff
@@ -0,0 +1,18 @@
+<div fxLayout="row wrap">
+  <mat-card  class="card" style="margin: 10px;" *ngFor="let unit of units"
+             fxFlex="30"
+             fxFlex.md="45"
+             fxFlex.sm="100"
+             fxFlex.xs="100"
+             [routerLink]="'/course/' + course.id + '/unit/' + unit.id">
```

Comment:
This should be the same component that points to a different URL based on the role. No need to change it at this point, but we should research how to restructure the front-end app.